Libraries/LibWeb/IndexedDB/Internal/Algorithms.cpp

```diff
@@ -1869,4 +1870,32 @@ WebIDL::ExceptionOr<JS::Value> retrieve_a_key_from_an_object_store(JS::Realm& re
     return convert_a_key_to_a_value(realm, record.value().key);
 }
 
+// https://w3c.github.io/IndexedDB/#retrieve-multiple-values-from-an-object-store
+WebIDL::ExceptionOr<JS::Value> retrieve_multiple_values_from_an_object_store(JS::Realm& realm, GC::Ref<ObjectStore> store, GC::Ref<IDBKeyRange> range, Optional<WebIDL::UnsignedLong> count_parameter)
+{
+    // 1. If count is not given or is 0 (zero), let count be infinity.
+    size_t count = count_parameter.has_value() ? count_parameter.value() : AK::NumericLimits<WebIDL::UnsignedLong>::max();
```

Comment:
This probably doesn't really matter functionally, but `count` is a `size_t` in `first_n_in_range`, so I suppose might be best to use size_t max here or even continue using OptionalNone as the sentinel value for infinity